src/ui/auth/ldap/ldap.go

```diff
@@ -84,8 +85,16 @@ func (l *Auth) Authenticate(m models.AuthModel) (*models.User, error) {
 		return nil, auth.NewErrAuth(err.Error())
 	}
 
+	//Get group admin dn
```

Comment:
I feel this chunk which populates the group info of a user should be moved to `postAuthenticate`?